core-helm-chart/values.yaml

```diff
@@ -2,6 +2,12 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+# Space abstraction layer image parameters
+image:
+  repository: ghcr.io/kubestellar/kubestellar/kubestellar-operator
+  pullPolicy: IfNotPresent
+  tag: 0.20.0-alpha.1
+
```

Comment:
We had several discussions in the release process doc with @MikeSpreitzer  about avoiding self references in git to a specific tag - this is something we were trying to avoid.

Reply:
I don't understand how is keeping the version in the Makefile better than keeping it in the helm template file?  But I updating the code to follow the existing convention.